demos/mkdocs/appyx-interactions/interactions/observemp/web/src/jsMain/kotlin/com.bumble.appyx.demos.observemp/ObserveMotionPropertiesSample.kt

```diff
@@ -105,10 +105,11 @@ fun <InteractionTarget : Any> ModalUi(
                 fontSize = 12.sp,
                 color = Color.White
             )
-            val dpOffset = motionPropertyRenderValue<Position.Value, Position>()?.offset
-            if (dpOffset != null) {
+            val alignment =
+                motionPropertyRenderValue<PositionOutside.Value, PositionOutside>()?.alignment
+            if (alignment != null) {
                 Text(
-                    text = "${roundFloatToTwoDecimals(dpOffset.x.value)}.dp",
+                    text = "Screens offset: ${roundFloatToTwoDecimals(alignment.horizontalBias)}",
```

Comment:
Typo

Reply:
Also the text is a bit misleading, since it's not the screen that's being offset, but the element.